.github/workflows/native.yml

```diff
@@ -12,15 +12,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macOS-latest, ubuntu-latest, windows-2019]
+        os: [macOS-latest, ubuntu-latest, ubuntu-22.04, windows-2019]
         include:
-          # Replace "example" with the name your binary
           - os: macOS-latest
             uploaded_filename: sbtn-x86_64-apple-darwin
             local_path: client/target/bin/sbtn
           - os: ubuntu-latest
             uploaded_filename: sbtn-x86_64-pc-linux
             local_path: client/target/bin/sbtn
+          - os: ubuntu-22.04
+            uploaded_filename: sbtn-aarch64-pc-linux
+            local_path: client/target/bin/sbtn
```

Comment:
`ubuntu-22.04` and `ubuntu-latest` refer to the same image.
However, adding `ubuntu-22.04` is just a stupid workaround so we kick off a extra job that generates us the aarch64 binary.

Reply:
You could add jobtype like https://www.scala-sbt.org/1.x/docs/GitHub-Actions-with-sbt.html#Build+matrix

Reply:
Hmm.. I thought I tried that, but it didn't work immediately, so I just use the now merged approach... Anyway, not so important I guess.